lib/Controller/AttachmentsController.php

```diff
@@ -152,9 +152,9 @@ public function destroy(string|int $id, ObjectService $objectService): JSONRespo
     {
         // Delete the attachment object
         $result = $this->objectService->deleteObject('attachment', $id);
-        
+
         // Return the result as a JSON response
-        return new JSONResponse(['success' => $result]);
+		return new JSONResponse(['success' => $result], $result === true ? '200' : '404');
```

Comment:
What happened here?

Reply:
https://conductionworkspace.slack.com/archives/C0378UVACUT/p1729007992160419?thread_ts=1729006242.757349&cid=C0378UVACUT

Reply:
[Fix delete in objectService](https://github.com/ConductionNL/opencatalogi/pull/98/commits/236c4a897f73dadc2d402a82eca5340cce9f6554)

[Fix delete in objectService 2.0](https://github.com/ConductionNL/opencatalogi/pull/98/commits/f10d2bcab9333936f7d445c8e99048b51f460013)

![image](https://github.com/user-attachments/assets/a3d4b528-e337-42a0-8980-a2ff69496384)